nifi/client.go

```diff
@@ -673,7 +673,7 @@ type RemoteProcessGroupComponent struct {
 	Name              string   `json:"name"`
 	Position          Position `json:"position"`
 	TargetUris        string   `json:"targetUris"`
-	TransportProtocol string   `json:"transportProtocol"`
+	TransportProtocol string   `json:"transport_protocol"`
```

Comment:
This should probably stay the way it was as this is used to generate/parse JSON blob sent to NiFi API. 
The fix applied below (in `nifi/resource_remote_process_group.go`) is what resolves the issue.